packages/app/src/components/inspector/ColorPickerGradient.tsx

```diff
@@ -176,21 +206,28 @@ export default memo(function ColorPickerSwatches({
         </Row>
       </PaddedSection>
       <PaddedSection>
-        {gradientLayout === 'grid' ? (
-          <SwatchesGrid
-            selectedSwatchId={swatchID}
-            gradients={gradientAssets}
-            gradientType={gradientType}
-            onSelectGradientAsset={onChange}
-          />
-        ) : (
-          <SwatchesList
-            selectedSwatchId={swatchID}
-            gradients={gradientAssets}
-            gradientType={gradientType}
-            onSelectGradientAsset={onChange}
-          />
-        )}
+        <ContextMenu.Root<MenuItemType>
+          items={menuItems}
+          onSelect={handleSelectMenuItem}
+        >
+          {gradientLayout === 'grid' ? (
+            <SwatchesGrid
+              gradients={gradientAssets}
+              gradientType={gradientType}
+              setGradientId={setGradientId}
+              onContextMenu={() => {}}
```

Comment:
For some reason if I don't add this, the contextMenu doesn't show at all😅

Reply:
I'm not sure, but I think we want `<ContextMenu.Root<MenuItemType>` around the individual `<Square />` components. It's a bit counterintuitive but for our ListView and GridView, we have a separate `<ContextMenu.Root<MenuItemType>` for each item, so we should be consistent with that approach. Let's try that first and see if there are still issues